app/lib/actions/katello/repository/index_yum_metadata_files.rb

```diff
@@ -0,0 +1,16 @@
+module Actions
+  module Katello
+    module Repository
+      class IndexYumMetadataFiles < Actions::EntryAction
+        def plan(repository)
+          plan_self(:user_id => ::User.current.id, :id => repository.id)
+        end
+
+        def run
+          repo = ::Katello::Repository.find(input[:id])
+          ::Katello::YumMetadataFile.import_for_repository(repo)
+        end
+      end
+    end
+  end
+end
```

Comment:
is this action needed?  I don't see it used

Reply:
addressed.